packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -110,7 +135,7 @@ export class RefreshPicker extends PureComponent<Props> {
           onClick={onRefresh}
           variant={variant}
           icon={isLoading ? 'spinner' : 'sync'}
-          style={width ? { width } : undefined}
+          style={width && isWideScreen ? { width } : undefined}
```

Comment:
I think you should be fine if you just check if `text` is empty here instead.
```suggestion
          style={width && text !== '' ? { width } : undefined}
```

Reply:
thanks for your review. but i don't think it works. because the `text` still exist. just it's css is `block: none`.
just like below image.
![image](https://github.com/user-attachments/assets/712e281c-134d-4e7e-8114-01ad1fd7a516)

Reply:
I looked into this a bit more. The issue is that the width is set specifically in Scenes. I'm looking into if there is any reason why this is needed. https://github.com/grafana/scenes/blob/2c5ef4771de511d7d0230a083b798699bbb9e581/packages/scenes/src/components/SceneRefreshPicker.tsx#L215